src/main/java/mate/academy/service/UserService.java

```diff
@@ -15,6 +15,16 @@ public class UserService {
      * Return <code>null</code> if there is no suitable user
      */
     public User findByEmail(String email) {
-        return null;
+
+        User userResult = new User("","");
+
+        for (int i = 0;i < users.length;i++) {
+            if (email.equals(users[i].getEmail())) {
+                userResult = users[i];
+            }
+        }
+
+        return userResult;
+
```

Comment:
read comment with task description carefully and make changes to your code
also use for each loop here